README.md

```diff
@@ -108,6 +108,7 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
 
   Support | Name | Description
   -- | -- | --
+   ! 3️⃣ ❕ | [ChartJs for Blazor](https://github.com/erossini/BlazorChartjs) | Blazor component to use ChartJs in Blazor applications
```

Comment:
- first emoji is incorrect, should be ❕ (not !)
- name should be `BlazorChartjs` (not "ChartJs for Blazor")
- description could be just "Blazor" (as we have for React, Vue.js, Java, etc.